app/views/kaui/invoices/index.html.erb

```diff
@@ -63,5 +70,22 @@ $(document).ready(function() {
       $(".dataTables_info").toggle(!noMoreData);
     });
   <% end %>
+
+  function searchInvoices () {
+    var invoiceSearch = document.getElementById('invoice-search').value;
+    table.search( invoiceSearch ).draw();
```

Comment:
* Is this performing only client-side search (in datatables)? What happens when there are multiple pages of invoices?
* Is this performing a full table search or just search on a specific column? Would search for "15" match both the invoice number and the invoice amount?

Reply:
Hi Pierre,

1. We will be doing client-side search only as we are utilizing Datatable [Search() API](https://datatables.net/reference/api/search()). Also, we will search from multiple pages & not a single column.
2. We are performing only a Number column search and not a full table search. So if we are searching for 15 it will search results only on invoice number and not other columns